mypy/checkexpr.py

```diff
@@ -250,6 +250,11 @@ def check_call(self, callee: Type, args: List[Node],
             self.check_argument_types(arg_types, arg_kinds, callee,
                                       formal_to_actual, context,
                                       messages=arg_messages)
+
+            if callee.is_type_obj() and (len(arg_types) == 1) and \
```

Comment:
The PEP 8 conformant way to break up such a line is
```
if (<long condition> and
        <another long condition> and
        <you get the point>):
    <body>
```
Note that you have to indent the continuations an extra 4 spaces or you'll run afoul of some other PEP 8 recommendation.

(Following some debate, it's also okay to put the `and` at the start of the line -- mypy is inconsistent about this anyways, so do what you think is best or what you see nearby. :-)